src/components/AddComment.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({ article_id, setComments }) => {
+  const [newAuthor, setNewAuthor] = useState("");
+  const [newBody, setNewbody] = useState("")
+    const [error, setErr] = useState(null)
+  
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
Please could you add in a validation check so I can't post a comment with a blank body